package/docker/openmrs/Dockerfile

```diff
@@ -42,6 +42,9 @@ RUN apt-get update && apt-get install gettext-base
 COPY package/docker/openmrs/bahmni_startup.sh /usr/local/tomcat/
 RUN chmod +x /usr/local/tomcat/bahmni_startup.sh
 
+# About adding recent openmrs-runtime.properties file
+COPY package/resources/openmrs-runtime.properties /usr/local/tomcat/.OpenMRS/
```

Comment:
@rohit-yawalkar this should already be there in the env right ? why are we copying again ?

Reply:
Approving it as the default comes from the openmrs docker images and this requires overriding.